src/python/include/Subsampling_interface.h

```diff
@@ -99,6 +110,16 @@ std::vector<std::vector<double>> subsampling_sparsify_points_from_file(const std
   std::vector<std::vector<double>> points = off_reader.get_point_cloud();
   return subsampling_sparsify_points(points, min_squared_dist);
 }
+#else
+// ------ sparsify_point_set ------
+std::vector<std::vector<double>> subsampling_sparsify_points(const std::vector<std::vector<double>>&, double) {
+  return std::vector<std::vector<double>>{{}};
```

Comment:
I don't think you need double `{{}}`, a single `{}` should work. Actually just `return {};`. We could also have thrown the exception from here, but your version with GUDHI_SUBSAMPLING_USE_CGAL is ok.

Reply:
Done on 9a2ed4e7